printer_mods/Phalanx/Other-V2-Idlers/README.md

```diff
@@ -0,0 +1,55 @@
+# "Other" V2 Idlers
+ Alternative Idlers that allow the usage of "Full-Figured" toolheads without the 10mm loss of travel at the front corners of the bed.
+
+These idlers were developed with the goal to allow the usage of wider toolheads with the Voron V2, better align the tensioning screw to the centerline of the idler shaft, add guides to the parts to allow them to interlock and provide a robust tensioning system, all while streamlining the process of making fine adjustments to tensioning. 
+This design iteration has been running on my machine for 50+ hrs with no issues or print quality degradation.  
+
+The idler design is ambidextrous to allow simplified batch printing. All parts are converted to stl in optimal printing orientation with quantities specified per Voron standards: 
+
+![image](https://user-images.githubusercontent.com/65414930/123886695-a1619680-d915-11eb-8a33-212a3cb1b8ef.png)
```

Comment:
Please upload all images used in this README into a directory under your mod, e.g. `printer_mods/Phalanx/Other-V2-Idlers/images/`, and then update this README to reference those uploaded images.  For example, if this image is uploaded as `images/naming_standards.png`, then reference it in the README as

    ![Naming standards](images/naming_standards.png)